Tiltfile

```diff
@@ -58,12 +58,12 @@ namespace = cfg.get("namespace", "wormhole")
 gcpProject = cfg.get("gcpProject", "local-dev")
 bigTableKeyPath = cfg.get("bigTableKeyPath", "./event_database/devnet_key.json")
 webHost = cfg.get("webHost", "localhost")
-algorand = cfg.get("algorand", True)
+algorand = cfg.get("algorand", False)
 aptos = cfg.get("aptos", True)
-evm2 = cfg.get("evm2", True)
-solana = cfg.get("solana", True)
-terra_classic = cfg.get("terra_classic", True)
-terra2 = cfg.get("terra2", True)
+evm2 = cfg.get("evm2", False)
+solana = cfg.get("solana", False)
+terra_classic = cfg.get("terra_classic", False)
+terra2 = cfg.get("terra2", False)
```

Comment:
Fair to say you didn't mean to do any of this in this PR?

Reply:
I did this to make the tilt build time faster, but didn't end up using tilt at all to write/test the Aptos contracts. Will revert these.